examples/now.rs

```diff
@@ -0,0 +1,74 @@
+#![no_std]
+/**
```

Comment:
I am okay with having more complex examples but not with having many examples with unclear purposes. Your example is a good candidate to replace the current timer example.